Sources/JWT/Claims/ExpirationTimeClaim.swift

```diff
@@ -4,8 +4,8 @@ import Node
 public struct ExpirationTimeClaim: TimeBasedClaim {
     public static var name = "exp"
 
-    let createTimestamp: () -> Seconds
-    let leeway: Seconds
+    public let createTimestamp: () -> Seconds
+    public let leeway: Seconds
```

Comment:
I'm not sure about making `leeway` public because none of the protocols force this. Is it for the sake of consistency?